rfcs/accordians-component.md

```diff
@@ -0,0 +1,141 @@
+# Accordians Component
+
+## Usage
+
+```js
+import {Accordians, Root, Toggle} from './index';
+
+export default () => {
+  return (
+    <div>
+      <Accordians
+        accordians={[
+          {
+            title: 'First title',
+            content: <div>First content</div>
+          },
+          {
+            title: 'Second title',
+            content: <div>Second content</div>
+          }
+        ]}
+      />
+    </div>
+  );
+};
+````
+
+```js
+import {Accordians, Root, Toggle} from './index';
+import {styled} from '../styles/index';
+
+const CustomRoot = styled(Root, props => ({
+  textColor: 'red',
+}));
+
+const CustomToggle = styled(Toggle, props => ({
+  backgroundColor: props.$isExpanded ? 'green' : 'blue',
+}));
+
+export default () => {
+  return (
+    <div>
+      <Accordians
+        onToggle={(event, index, isExpanded) => {console.log('Accordian ' + index + ' is ' + (isExpanded ? 'expanded' : 'collapsed') )}}
+        overrides={{
+          Root: CustomRoot,
+          Toggle: CustomToggle,
+        }}
+        accordians={[
+          {
+            title: 'First title',
+            content: <div>First content</div>
+          },
+          {
+            title: 'Second title',
+            content: <div>Second content</div>
+          },
+          {
+            title: 'Third title',
+            content: <div>Third content</div>
+          }
+        ]}
+        expanded={[false, true]}
+        autoExpand={true}
+      />
+    </div>
+  );
+};
+```
+
+## Exports
+
+* `Accordians`
+* `StyledRoot`
+* `StyledToggle`
+* `StyledTitle`
+* `StyledContent`
+
+## `Accordians` API
+
+* `accordians: Array<Item>` - Required
+  All accordians in component. Each `Item` has:
+  * `title: React$Node` - title of accordian
+  * `content: React$Node` - content of accordian
+* `autoExpand: boolean` - Optional
+  If set to true, it will auto expand next accordian right after current is collapsed. Default is `'false`
```

Comment:
I don't know if I've ever seen this behavior in an accordion before? I've see the case where opening the _next_ accordion closes the previous (which I called `exclusive` in the code example above), but not the case you're describing here.

Reply:
yes, this is what i meant

Reply:
renamed to exclusive